lib/ui/screens/redeem_coin/redeem_coin_page.dart

```diff
@@ -89,9 +89,12 @@ class _RedeemCoinPageState extends ConsumerState<RedeemCoinPage> {
                               context,
                               child: AccountSelectionPage(
                                 onTapRedeemCoin: (String address) async {
-                                  CustomAlertWidget.loader(true, context);
-                                  await ref.read(redeemCoinViewModelProvider).redeemCoin(assetId: redeemCoinModel.assetId, addr: address, context: context);
-                                  CustomAlertWidget.loader(false, context);
+                                  if (!showCoinLoaderIds.value.contains(redeemCoinModel.assetId)) {
```

Comment:
is this for the async redeeming? i thought thats its #731 ~ or is this something else?

Reply:
Yes here we show loader on redeem coin tile.

Reply:
so the PRs are mixed?